### DISCUSSION TITLE
Improve icon

### DISCUSSION THREAD

**Discussion by tobimensch** [iSoron/uhabits#38] [Feature Requests]
The current icon looks too unspecific and boring in my opinion. I'm not really fond of the blue color, maybe green would be better, but that's just my personal taste.

Unfortunately I'm not great at making graphics, but here are a couple of ideas for a nice icon:
- Left/-right brain hemispheres, where one side is overlayed with a "checked" symbol and one is overlayed with a "unchecked" symbol
- Spiral of "checked" symbols from the center to the outside, the last outer symbol(s) on the spiral
  should however be "unchecked" indicating that there's still something to be done
- Cigarette pack with multiple unchecked/checked symbols on the front (play on the idea of habits forming = getting addicted)
- Cigarette pack(or something similar) with the label "Habits"/(or "Loop Habits") on it

I think many people will have ideas for a better icon, these are just the results of my brainstorming.

**Comment by iSoron** [comment]
I am not planning on designing a new icon any time soon, but if anyone is a designer, please feel free to send some suggested icons. Keep in mind that the icon should be very simple and should follow the [material design guidelines](https://www.google.com/design/spec/style/icons.html).

**Comment by c33s** [comment]
i like the current icon, it is straight and simple.

changing the color would be ok for me, but this is mostly because of many the apps i use use the color blue :D

**Comment by trymeouteh** [comment]
I think the current icon is perfect. Simple and clean. I am picky when it comes to icons and I think this is the best Habit Tracking app icon out there.
